examples/resources/awscc_opensearchserverless_access_policy/access_policy.tf

```
@@ -0,0 +1,29 @@
+resource "awscc_opensearchserverless_access_policy" "os" {
+  name = "test-os-security-policy"
+  policy = jsonencode([{
+    "Description" = "Access for test-user",
+    "Rules" = [
+      {
+        "ResourceType" = "index",
+        "Resource" = [
+          "index/*/*"
+        ],
+        "Permission" = [
+          "aoss:*"
+        ]
+        }, {
+        "ResourceType" = "collection",
```

Let's pretty this up 

```suggestion
      {
        "ResourceType" = "index",
        "Resource" = [
          "index/*/*"
        ],
        "Permission" = [
          "aoss:*"
        ]
      },
      {
        "ResourceType" = "collection",
```